impl/src/main/java/org/jboss/weld/logging/BootstrapLogger.java

```diff
@@ -194,4 +194,8 @@
     @LogMessage(level = Logger.Level.DEBUG)
     @Message(id = 145, value = "Bean identifier index built:\n  {0}", format = Format.MESSAGE_FORMAT)
     void beanIdentifierIndexBuilt(Object info);
+
+    @LogMessage(level = Level.WARN)
+    @Message(id = 146, value = "Method BeforeBeanDiscovery.addAnnotatedType(AnnotatedType<?>) is deprecated!\n", format = Format.MESSAGE_FORMAT)
```

Comment:
Please remove the `\n` char. Maybe we should also add some more info, e.g. the underlying java class of the given AT.